src/main/config/thurloe.conf.ctmpl

```diff
@@ -0,0 +1,22 @@
+{{with $environment := env "ENVIRONMENT"}}{{$keyname := printf "secret/dsde/%s/thurloe/thurloe.conf" $environment}}{{with vault $keyname}}
+
+include "application.conf"
+
+database {
+  config = {{.Data.db_config}}
+
+  main {
+    google-dev {
+      db.url = "{{.Data.db_url}}"
+      db.user = "{{.Data.db_user}}"
+      db.password = "{{.Data.db_password}}"
+      db.driver = "{{.Data.db_driver}}"
+      db.connectionTimeout = 5000
+      driver = "{{.Data.driver}}"
+      connectionPool = {{.Data.connection_pool}}
+    }
+  }
+}
+
```

Comment:
Add in the crypto stanza:
crypto {
  key = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
}